content/blog/2021-12-02-collaborative-experiments.md

```diff
@@ -0,0 +1,163 @@
+follow along with a project you're currently working on, make sure to check out
+the steps to initialize a DVC pipeline in
+[the Getting Started doc](https://dvc.org/doc/start).
+
+## Setting up your DVC remotes
+
+When you want to share the progress you've made with training your model, that
+usually means you need to find a way to bundle the code, data, and
+hyperparameters. This could be a complicated process if you're working with GBs
+worth of data or you have a large number of hyperparameters.
+
+That's one of the uses for DVC and why we'll be working with remotes. To start
+with, make sure your GitHub remote is configured correctly. It should use the
+SSH version of the URL. This is so DVC can authenticate the pushes and pulls
+from GitHub it needs as part of experiment sharing.
+
+The way DVC works is by storing custom Git refs in your repo with metadata that
+defines the experiment. You can learn more about how DVC uses custom Git refs in
+[this post](https://dvc.org/blog/experiment-refs).
+
+Next, you'll need to set up a remote to your data location. This could be an AWS
+S3 bucket, a Google Drive, or
+[one of the other supported storage types](https://dvc.org/doc/command-reference/remote/add#supported-storage-types).
+
+An important thing to note about the project we're working with is that there is
+already a remote set up for you to pull from. You can see this in `.dvc/config`.
+You'll need to set up a separate remote to push changes to since this remote
+doesn't allow push access.
+
+For this example, we'll be using a Google Drive folder as the remote to handle
+pushing data. Now that you know what we're doing, let's run the command to set
+up the DVC remote to push to.
+
+```dvc
+$ dvc remote add cloud_remote gdrive://1k6aUYWphOulJlXgq4XbfKExWGyymTpEl
+```
+
+This adds the remote storage named `cloud_remote` for DVC to track and we'll be
+able to push and pull the exact code and data to reproduce any experiment. With
+your Git remote and DVC remotes in place, you can start pulling data and
+experiments from the cloud to your local machine.
+
+_Note: Make sure you have write permissions to the Git remote!_
+
+## Listing your remote experiments
+
+When you're working with a team on an existing project, you might want to see
+the experiments already in the remotes so you know what's available. To take a
+look at the experiments we have run in the repo you forked, you'll have to set
+up a new Git upsteam remote to reference the original repo. You can do that with
+the following command.
+
+```dvc
+$ git remote add upstream https://github.com/iterative/example-dvc-experiment
+```
+
+Now you can take a look at all of the experiments we have associated with this
+repo with the following command.
+
+```dvc
+$ dvc exp list upstream --all
+```
+
+You'll get a list of all of the experiments across different Git branches that
+have been pushed with DVC in the original repo. The output will look similar to
+this.
```

Comment:
We may explicity note that the experiment names might be changed, but "similar to" may also be enough. Thank you.

Reply:
I think I'll leave it as "similar to" so we can finish this post. 🙂